gradle.properties

```diff
@@ -2,7 +2,7 @@
 kotlin.code.style=official
 
 # config
-version=1.4.0
+version=1.4.1-SNAPSHOT
```

Comment:
Most likely commit "KTOR-1027 Update versions" shouldn't be here

Reply:
rebased